library.json

```diff
@@ -31,6 +31,7 @@
             "googlemock/CMakeLists.txt",
             "googlemock/Makefile.am",
             "googlemock/configure.ac",
+            "googlemock/src/gmock_main.cc",
```

Comment:
looks weird to read, could you please sort alphabetically

Reply:
done